### ISSUE TITLE
Give Microsub server access to private posts for its user?

### ISSUE THREAD

**Issue by aaronpk** [indieweb/microsub#1]
When a website adds https://indieweb.org/private_posts within feeds for auth’d visitors, it could be important for the Microsub server to be able to fetch these posts if the current user is included in the https://indieweb.org/audience.

**Comment by sknebel** [comment]
Agreed, this is a piece of the puzzle we've been circling. Another purpose coming to mind are translation services like (recently discussed in #indieweb) an e-mail-to-feed bridge, which shouldn't have to be part of the microsub server directly, but still be secure.

**Comment by dshanske** [comment]
See: https://indieweb.org/IndieAuth_Ticket_Auth as a way being explored for this.
